src/test/resources/com/puppycrawl/tools/checkstyle/main/InputMainComplexityOverflow.java

```diff
@@ -7,159 +7,160 @@
 public class InputMainComplexityOverflow {
 
   public void provokeNpathIntegerOverflow()
-  {
-    if (true) {
+    {
```

Comment:
About these changes:
This file is checked against google_checks.xml, so earlier it was not following the correct indentation proposed by google. Now it does, basically this change is important for ommiting long lines of indentation x-path suppression script.

Reply:
confirmed, wihtout this update, we need to add 2 more suppression in expect output of [MainTest.java](https://github.com/checkstyle/checkstyle/blob/7a71e8ae7a746f8da6262fe03a137ed4a328e4bd/src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java#L1081)